src/modules/mentors/mentors.controller.ts

```diff
@@ -39,20 +39,27 @@ export class MentorsController {
   @Get()
   @UsePipes(new PaginationPipe())
   async index(@Req() request: Request, @Query() filters: MentorFiltersDto) {
-    const data = await this.mentorsService.findAll(filters, !!request.user);
+    const available = await this.mentorsService.findAll({ ...filters, available: true }, !!request.user);
```

Comment:
That totally not my strong side but isn't better to call once to the database and do the filter (split to available and unavailable) in the code?

Reply:
Given that we only have about 500 mentors and that later on we are planning to support pagination, this makes sense. 

I was originally thinking on avoiding looping though thousands of records, but that won't be case in the long term.